### PR TITLE
Improve error when command is not defined

### PR DESCRIPTION
**Pull request by thiagoa**
This pull request is part of the following issue: rom-rb/rom#324

I made the following changes:

- Removed `ROM::Registry` tests, which were moved over to the `rom-support` repo, in the following pull request: https://github.com/rom-rb/rom-support/pull/6
- Added tests for the new "default value" feature of `ROM::Registry#fetch`. The code is in `rom-support` (PR referenced above), but the tests are in this repository. To make this possible I temporarily pointed the `rom-support` gem reference to my `rom-support` fork, but this will get removed once the related pull request is OK.
- I'm using the "default value" feature to throw a specialized error in case a command is not found under a relation.

### CODE REVIEW

**Comment by solnic** [comment]
Thanks for this. The specs should be moved to rom-support. Here we could only verify if command-specific registry has correct behavior.

**Comment by thiagoa** [comment]
@solnic Thanks. I've just updated this pull request with the removal of the registry spec file, and the rom-support pull request was updated with the same spec file which was moved over from here.

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'rom-support', github: 'thiagoa/rom-support', branch: 'add-support-for-default-value-in-rom-registry-fetch'
```
Comment (solnic) [inline Gemfile]: This can be changed to `rom-rb/rom-support` and branch `master` now

**Comment by solnic** [comment]
Could you rebase on top of master so that we can get a clean merge?

**Comment by thiagoa** [comment]
@solnic All done

**Pull request merged**